data/map.txt

```diff
@@ -26130,6 +26148,15 @@ planet Pearl
 		threshold 6000
 		fleet "Large Republic" 11
 
+planet "Pelubta Station"
+	attributes arach "coalition station" research
+	landscape land/station5
+	description `Pelubta Station was built shortly after the Arach discovered the hyperdrive, initially to take advantage of the massive amounts of solar power generated by the nearby stars. However, now it is mainly used as a hyperspace fuel refinery. Although it has endured some solar flares, constant maintenance means that the station still doesn't show it's age.`
+	description `	Pelubta Station is also used to study one of the binary stars in this system, since it has a similar makeup and age to one orbited by a Heliarch ringworld. Most of this research is focused on predicting stellar phenomena, expecially the most abnormal ones.`
+	spaceport `The intake section has a consistently high traffic, mostly made up of Coalition merchants supplying the station and leaving with shipments of fuel to take to other Arach worlds.`
+	spaceport `	The science section is closed off to non-Heliarchs. Considering that the work done here should be relatively benign, the station may also serve other purposes, such as development of superweapons or observing the nearby Human space.`
```

Comment:
Could probably just make this all one paragraph since they're so short.
Human isn't capitalized.
```suggestion
	spaceport `	The science section is closed off to non-Heliarchs. Considering that the work done here should be relatively benign, the station may also serve other purposes, such as development of superweapons or observing the nearby human space.`
```
This also feels a bit on the nose. Perhaps remove the speculation? i.e. don't say what it is that they could be doing.